pkg/registry/controller/etcd/etcd_test.go

```diff
@@ -319,6 +314,21 @@ func TestEtcdGetController(t *testing.T) {
 	test.TestGet(&copy)
 }
 
+func TestEtcdListControllers(t *testing.T) {
+	storage, fakeClient := newStorage(t)
```

Comment:
I assume that we plan to do the same for Create, Delete and other operations as well in future PRs.